ci/build_and_tag_images.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail
```

We should add some of these flags to catch errors.

We should also consider standardizing which flags we use across all files.

Looks like there are some inconsistencies currently.

Reply:
I've updated this throughout the repo.